com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -4,6 +4,13 @@
 
 namespace UnityEngine.Rendering.HighDefinition
 {
+    internal enum FPTLMaxLightSizes
+    {
+        Small = 23,
```

Comment:
I don't see any reasons to have this number configurable if there is no impact on perf. 63 is fine. If this is related to memory saving then we should evaluate the memory cost but then there is no difference between 31 and 63 so we should remove 23

Reply:
As discussed, we want to keep this flexible so the user can do intense 8k rendering with a high light count cluster (>128 lights on a single spot).

Reply:
note. ok for 31 and 63 but 23 should be remove

Reply:
Removed 23